src/services/langfuse.py

```diff
@@ -51,26 +59,43 @@ class LangfuseService(metaclass=SingletonMeta):
         public_key (str): The public key used for authentication, set to LANGFUSE_PUBLIC_KEY.
         secret_key (str): The secret key used for authentication, set to LANGFUSE_SECRET_KEY.
         auth (BasicAuth): An authentication object created using the public and secret keys.
+        debug_enabled (bool): A boolean flag to enable debug mode, set to LANGFUSE_DEBUG_MODE.
         _handler (CallbackHandler) : A callback handler for langfuse.
     """
 
     def __init__(self):
-        self.base_url = LANGFUSE_HOST
-        self.public_key = LANGFUSE_PUBLIC_KEY
-        self.secret_key = LANGFUSE_SECRET_KEY
+        self.base_url = str(LANGFUSE_HOST)
+        self.public_key = str(LANGFUSE_PUBLIC_KEY)
+        self.secret_key = str(LANGFUSE_SECRET_KEY)
         self.auth = BasicAuth(self.public_key, self.secret_key)
         self._handler = CallbackHandler(
-            secret_key=LANGFUSE_SECRET_KEY,
-            public_key=LANGFUSE_PUBLIC_KEY,
-            host=LANGFUSE_HOST,
-            enabled=string_to_bool(LANGFUSE_ENABLED),
+            secret_key=str(LANGFUSE_SECRET_KEY),
+            public_key=str(LANGFUSE_PUBLIC_KEY),
```

Comment:
as you did for the host:
```suggestion
            secret_key=self.secret_key,
            public_key=self.public_key,
```